firecrown/updatable.py

```diff
@@ -57,14 +57,22 @@ class Updatable(ABC):
 
     """
 
-    def __init__(self) -> None:
-        """Updatable initialization."""
+    def __init__(self, parameter_prefix: Optional[str] = None) -> None:
+        """Updatable initialization.
+
+        :param prefix: prefix for all parameters in this Updatable
```

Comment:
We should fix the documentation so that it matches the argument name “parameter_prefix” (or change the code to match the comment).